ui/src/app/modules/logical-group/components/logical-group.component.html

```diff
@@ -13,7 +13,7 @@ <h3>Custom Groups</h3>
     Last Month Costs
 </button>
 <button type="button" (click)="changeLast3MonthView()" class="label clickable {{ getLast3MonthStatus() }}" style="cursor:pointer;font-size: 10pt;">
-    Last 3 Months Costs
+    3 Months Costs
```

Comment:
Also, good to have a signpost here - https://vmware.github.io/clarity/documentation/v0.11/signposts to explain what is shown.

Reply:
Done